modules/exploits/linux/http/netgear_r7000_cgibin_exec.rb

```diff
@@ -40,7 +40,8 @@ def initialize(info = {})
           'Space'       => 1024,
           'DisableNops' => true,
           'BadChars'    => "\x20"
-        }
+        },
+      'DefaultOptions'  => { 'WfsDelay' => 10}
```

Comment:
I have yet to do a full badchar analysis, too. Will probably have to start reversing bins.